Cache/IO.lean

```diff
@@ -117,6 +127,7 @@ private def CacheM.mathlibDepPath (sp : SearchPath) : IO FilePath := do
   return mathlibSource
 
 -- TODO this should be generated automatically from the information in `lakefile.lean`.
+@[inherit_doc CacheM.Context]
```

Comment:
This seems a little much to me, the docstring for the type doesn't really belong on its constructor as well. This is private anyway, so I think can be left undocumented.
```suggestion
```